Fedoraware/Fedoraware-TF2/src/SDK/Main/BaseEntity/BaseEntity.h

```diff
@@ -9,7 +9,7 @@
 	return GetVFunc<fn>(pBase, index)(pBase); \
 }
 
-#define HAS_CONDITION(ent, cond) (ent->GetCond() & cond)
+#define HAS_CONDITION(ent, cond) (ent->InCond(cond))
```

Comment:
This macro seems useless now, since it simply refers to the InCond function. Can you remove it and replace its usages with the function call?